### PR TITLE
feat(platform/ci): Add prod migrations

### PR DESCRIPTION
**Pull request by aarushik93**
### Background

So prod migrations run automatically

### Changes 🏗️

Added migrate job

Only for the new autogpt platform, currently in autogpt_platform/

- Create from scratch and execute an agent with at least 3 blocks
- Import an agent from file upload, and confirm it executes correctly
- Upload agent to marketplace
- Import an agent from marketplace and confirm it executes correctly
- Edit an agent from monitor, and confirm it executes correctly

### CODE REVIEW

File: .github/workflows/platform-autgpt-deploy-prod.yml [hunk 1]
```diff
@@ -15,6 +15,49 @@ env:
...
+        env:
+          DATABASE_URL: ${{ secrets.BACKEND_DATABASE_URL }}
+
+      - name: Run Market Migrations
+        if: steps.check_changes.outputs.market_changed == 'true'
+        working-directory: ./autogpt_platform/market
+        run: |
+          python -m prisma migrate deploy
+        env:
+          DATABASE_URL: ${{ secrets.MARKET_DATABASE_URL }}
```
Comment (ntindle) [inline .github/workflows/platform-autgpt-deploy-prod.yml]: is there a reason we wouldn't always run migrations?

**Review by ntindle — DISMISSED** [review]

File: .github/workflows/platform-autgpt-deploy-prod.yml (hunk 1 above)
Comment (aarushik93) [reply .github/workflows/platform-autgpt-deploy-prod.yml]: when there are no changes in the migrations?

Comment (ntindle) [reply .github/workflows/platform-autgpt-deploy-prod.yml]: Doesn’t the tool determine that?

Comment (ntindle) [reply .github/workflows/platform-autgpt-deploy-prod.yml]: If I'm crazy feel free to resolve btw

Comment (aarushik93) [reply .github/workflows/platform-autgpt-deploy-prod.yml]: fair i was thinking we just don't do this step at all thats why. but ill update!

**Pull request merged**